openvdb_points/tools/AttributeSet.h

```diff
@@ -155,9 +155,6 @@ class AttributeSet
     ///         shared with anyone else.
     void makeUnique(size_t pos);
 
-    /// Copy attribute values for a target index @a n from another @a attributeSet for index @a source
```

Comment:
I agree with removing this method, there's currently no functionality that uses it.